### PR TITLE
Replace images on playground steps

### PR DESCRIPTION
**Pull request by borisvida**

### CODE REVIEW

File: client/src/components/Playground/Playground.js [hunk 1]
```diff
@@ -339,6 +342,12 @@ const Playground = ({history, typingDone, setTypingDone}) => (
         <div className="footer-text">
           Backround svg - Free Vector Graphics by <a href="https://www.vecteezy.com">Vecteezy!</a>
         </div>
+        <div className="footer-text">
+          Icons made by <a href="https://www.freepik.com/" title="Freepik">Freepik</a> from
```
Comment (tomasfabry) [inline client/src/components/Playground/Playground.js]: Now I've noticed one thing - there are missing spaces between texts and links. But I really don't know why:
image

**Comment by tomasfabry** [comment]
I've also got some warnings:
```./src/components/Playground/Playground.js
  Line 131:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 259:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 274:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 289:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 304:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 319:  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 334:   Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
  Line 349:   Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
```

I know it's not due to your changes, but maybe you can fix it if you will make some fixes related to the footer...

**Comment by lenhardt** [comment]
I also noticed that I need to add {' '} if I want to have space before or after <a/>.

Sorry for the errors with "_blank" .. I will see how you fix it and will try not to introduce new errors like that!

**Comment by borisvida** [comment]
@tomasfabry Fixed, please, re-check :slightly_smiling_face: 

@lenhardt About that `_blank` errors: you just need to add `rel="noopener noreferrer"` to the anchor. It's for security reasons, summed up here: https://mathiasbynens.github.io/rel-noopener/

**Pull request merged**